src/pretix/presale/templates/pretixpresale/event/fragment_checkoutflow.html

```diff
@@ -5,9 +5,16 @@
             <div class="checkout-step-bar-left"></div>
             <div class="checkout-step-bar-right"></div>
             <div class="checkout-step-icon">
-                <span class="fa {% if step.c_is_before %}fa-check{% elif step.icon %}fa-{{ step.icon }}{% else %}fa-pencil{% endif %}"></span>
+                <span class="fa {% if step.c_is_before %}fa-check{% elif step.icon %}fa-{{ step.icon }}{% else %}fa-pencil{% endif %}" aria-hidden="true"></span>
             </div>
             <div class="checkout-step-label">
+                <span class="sr-only">
+                {% if step.c_is_before %}
+                    {% trans "Completed:" %}
+                {% elif request.resolver_match.kwargs.step == step.identifier %}
+                    {% trans "Current:" %}
+                {% endif %}
+                </span>
                 {{ step.label }}
             </div>
```

Comment:
Should this maybe become a `<nav>`?

Reply:
Hm, good question. I got this from here: https://www.w3.org/WAI/tutorials/forms/multi-page/ and they did not use a `<nav>`. I guess it’s not really a nav, as it is not meant nor possible to jump between steps but only go back?

Reply:
It is possible to jump backwards multiple steps, but yeah, it doesn't really matter to me